content/articles/2017-08-25-lucy-experience.md

```diff
@@ -0,0 +1,79 @@
+
+The googledrive 📦 looked measurably different on this [first day](https://github.com/tidyverse/googledrive/tree/ef8a410f8e74080670ff3145a330cdaa100472a8). I think this gives some really good insight into the iterative process of package design as well as how much I absorbed from Jenny along the way. In these early days, I was beavering away, building some small wrappers for Drive endpoints, spending most of my time getting to know the [Drive documentation](https://developers.google.com/drive/v3/web/about-sdk) and celebrating small victories.
+
+<div style="text-align: center;">
+<div style="display: inline-block; text-align: left">
+<img src = "/images/lucy-internship/02_upload.png" width = "300pt"> </img><br>_Small victory 1: The ability to upload files to Drive_.
+
+<img src = "/images/lucy-internship/03_delete.png" width = "300pt"> </img><br> _Small victory 2: The ability to delete files on Drive_.
+</div>
+</div>
+</br>
+
+My day to day back in April seems so foreign to the workflow Jenny and I have now established, but those first few weeks were incredibly useful for getting me steeped in all things Google Drive. Our process eventually converged to:
+
+👩🏻‍💻👩🏼‍💻 Video chatting once a week to set priorities / discuss design decisions.  
+✍️🙊 Documenting all problems as GitHub issues to resolve.  
+👷‍♀️👀 Submitting all changes via PRs for review. 
+
+The PR review process was quite new to me, but SO integral to the success of this package. Here is an [example](https://github.com/tidyverse/googledrive/pull/13) of what this can look like. Our first PR with review consisted of 49 commits and 66 comments 😱. The GitHub review process allowed Jenny to comment line by line on my code, gently pushing me towards better coding conventions and style. This process really helped me to absorb all things "tidyverse" in a systematic way.
+
+
+## _Absorbing all things "tidyverse"_ <br> <img src = "/images/lucy-internship/03_markdown-docs.png" width = "400pt"></img>
+
+<center>
+<blockquote class="twitter-tweet" data-lang="en"><p lang="en" dir="ltr">🌞 musing: learning a new coding style is like putting on freshly washed jeans; a struggle, but once I get it I&#39;m like that looks GOOD 💃 <a href="https://t.co/5QcEwEcI23">pic.twitter.com/5QcEwEcI23</a></p>&mdash; Lucy 🌻 (@LucyStats) <a href="https://twitter.com/LucyStats/status/873587893754843136">June 10, 2017</a></blockquote> <script async src="//platform.twitter.com/widgets.js" charset="utf-8"></script>
+</center>
+
+The tidyverse has an [R Style Guide](http://style.tidyverse.org). My first pass at incorporating the style was switching my roxygen documentation to [markdown](https://github.com/klutometis/roxygen/blob/master/vignettes/markdown.Rmd) (this can be enabled by adding `Roxygen: list(markdown = TRUE)` to the `DESCRIPTION` file). This is an example of something I didn't even know existed prior to this project. Other pieces I absorbed include: 
```

Comment:
Mention roxygen2md?

Reply:
😱 🙊 I didn't even know about that! I'll definitely add it.